vdoctl/cmd/drivers.go

```diff
@@ -0,0 +1,351 @@
+			fetchVCIP(&cpi, labels, "Cloud Provider")
+			vcIPList = append(vcIPList, cpi.vcIp)
+			if !cpi.insecure {
+				thumbprintMap[cpi.vcIp] = cpi.thumbprint
+			}
+
+		multivcloop:
+			for {
+
+				fetchCredentials(&cpi, labels, "Cloud Provider")
+
+				secret, err := createSecret(K8sClient, ctx, cpi, "cpi")
+				if err != nil {
+					panic(err)
+				}
+
+				vcc, err := createVsphereCloudConfig(K8sClient, ctx, cpi, secret.Name, "cpi")
+				if err != nil {
+					panic(err)
+				}
+				vsphereCloudConfigList = append(vsphereCloudConfigList, vcc.Name)
+
+				multiVC := utils.PromptGetInput("Do you want to configure another VC for CPI? (Y/N)", errors.New("invalid input"), utils.IsString)
+
+				if multiVC == "Y" || multiVC == "y" {
+					isCPIMultiVC = true
+					cpi = credentials{}
+
+					fetchVCIP(&cpi, labels, "Cloud Provider")
+					vcIPList = append(vcIPList, cpi.vcIp)
+					if !cpi.insecure {
+						thumbprintMap[cpi.vcIp] = cpi.thumbprint
+					}
+					continue multivcloop
+
+				}
+				cpi.vsphereCloudConfigs = vsphereCloudConfigList
+				break
+
+			}
+
+			topology := utils.PromptGetInput("Do you want to configure zones/regions for CPI? (Y/N)", errors.New("invalid input"), utils.IsString)
+
+			if topology == "Y" || topology == "y" {
+				cpi.topology.Zone = utils.PromptGetInput(labels.topology.Zone, errors.New("unable to get the zones"), utils.IsString)
+				cpi.topology.Region = utils.PromptGetInput(labels.topology.Region, errors.New("unable to get the regions"), utils.IsString)
+			}
+
+			fmt.Println("You have now completed configuration of Cloud Provider. We will now proceed to configure Storage Provider. ")
+
+		}
+
+		if isCPIMultiVC {
+			csi.vcIp = utils.PromptGetSelect(vcIPList, "Please select vcenter for configuring Storage Provider?")
+			if _, ok := thumbprintMap[csi.vcIp]; ok {
+				csi.thumbprint = thumbprintMap[csi.vcIp]
+				csi.insecure = false
+			} else {
+				csi.insecure = true
+			}
+
+		} else if isCPIRequired == "Y" || isCPIRequired == "y" {
+			csi.vcIp = cpi.vcIp
+			csi.insecure = cpi.insecure
+			csi.thumbprint = cpi.thumbprint
+		} else {
+			fetchVCIP(&csi, labels, "Storage Provider")
+		}
+
+		fetchCredentials(&csi, labels, "Storage Provider")
+
+		secret, err := createSecret(K8sClient, ctx, csi, "csi")
+
+		if err != nil {
+			panic(err)
+		}
+
+		vcc, err := createVsphereCloudConfig(K8sClient, ctx, csi, secret.Name, "csi")
+		if err != nil {
+			panic(err)
+		}
+		csi.vsphereCloudConfig = vcc.Name
+
+		advConfig := utils.PromptGetInput("Do you wish to configure File Volumes? (Y/N)", errors.New("invalid input"), utils.IsString)
+
+		if advConfig == "Y" || advConfig == "y" {
+
+			vsanDSurl := utils.PromptGetInput("vSANDataStoresUrl", errors.New("unable to get the vSANDataStoresUrl"), utils.IsString)
+			csi.vSANDataStoresUrl = strings.SplitAfter(vsanDSurl, ",")
+
+			netPerms := utils.PromptGetInput("Do you wish to configure netPermission? (Y/N)", errors.New("invalid input"), utils.IsString)
```

Comment:
can this say
"Do you wish to configure Net File permissions for File Volumes (Y/N)"